docs/website/docs/index.md

```diff
@@ -6,6 +6,12 @@ lowers Machine Learning (ML) models to a unified IR that scales up to meet the
 needs of the datacenter and down to satisfy the constraints and special
 considerations of mobile and edge deployments.
 
+## User Guide
+
+Use of IREE requires three main stages outlined in our 
+[User Getting Started Guide](./getting-started/index.md). Overall these stages
+can be broken into import, compilation, and execution of a model using IREE.
```

Comment:
I mainly wanted to include this at head to divert non-developers to the Getting Started guide as quickly as possible. What is your thought on moving the workflow to the top.

Reply:
Good point. Maybe worth splitting that sort of change into a separate PR.

Some ideas:
* We could move the "Project architecture" section down
* I think "Key features" and "Support matrix" are important to keep near the top so people can first learn what the project is and if it sounds useful, _then_ see how to use it
* The "Support matrix" section is a little _too_ sparse now... it could be condensed

Reply:
Overall having a toploaded section that is literally "Hey, if you just want to use things go here" with explanations remaining in the other section.